index.js

```diff
@@ -0,0 +1,74 @@
+const task = document.getElementById("add-task");
+const list = document.getElementById("list");
+const clearItemsbtn = document.querySelector(".clear-btn");
+
+var editFlag = false;
+var editElement;
+
+task.addEventListener("keypress", (e) => {
+    if (e.key == "Enter") {
+        addItemToList(task.value);
+        task.value = "";
+        if (list.childNodes.length >= 0) {
+
+            clearItemsbtn.classList.add("clear-btn-diplay")
+        }
+    }
+
+});
+
+
+function addItemToList(value) {
+
+    if (value == "") {
+        alert("No task to add");
```

Comment:
instead of showing alert create a div for err message and hide and show that div with appropriate message